test/functional/interface_rest.py

```diff
@@ -235,13 +234,13 @@ def run_test(self):
         response_hex = self.test_rest_request("/block/{}".format(bb_hash), req_type=ReqType.HEX, ret_type=RetType.OBJ)
         assert_greater_than(int(response_hex.getheader('content-length')), BLOCK_HEADER_SIZE*2)
         response_hex_bytes = response_hex.read().strip(b'\n')
-        assert_equal(binascii.hexlify(response_bytes), response_hex_bytes)
+        assert_equal(bytes.hex(response_bytes).encode(), response_hex_bytes)
```

Comment:
```suggestion
        assert_equal(response_bytes.hex().encode(), response_hex_bytes)
```

Is it possible to use the member method? (Didn't check myself)

Reply:
Yes, just tested on my end to verify, it has the same effect. Makes sense, as `bytes.hex(bytes_obj) == bytes_obj.hex()` and by extension `bytes_obj.hex().encode() == binascii.hexlify(bytes_obj)`.